go.mod

```diff
@@ -4,54 +4,61 @@ go 1.24.7
 
 require (
 	github.com/container-storage-interface/spec v1.6.0
-	github.com/google/go-cmp v0.5.9
+	github.com/google/go-cmp v0.7.0
 	github.com/kubernetes-csi/csi-lib-utils v0.10.0
 	github.com/kubernetes-csi/csi-test/v4 v4.3.0
-	github.com/onsi/gomega v1.27.4
-	github.com/prometheus/client_golang v1.15.0
-	github.com/stretchr/testify v1.8.2
-	go.opentelemetry.io/otel v1.15.1
-	go.opentelemetry.io/otel/exporters/prometheus v0.38.1
-	go.opentelemetry.io/otel/metric v0.38.1
-	google.golang.org/grpc v1.56.3
-	google.golang.org/protobuf v1.33.0
-	k8s.io/api v0.26.4
-	k8s.io/apimachinery v0.26.4
-	k8s.io/client-go v0.26.4
-	k8s.io/klog/v2 v2.100.1
+	github.com/onsi/gomega v1.32.0
+	github.com/prometheus/client_golang v1.23.0
+	github.com/stretchr/testify v1.11.1
+	go.opentelemetry.io/otel v1.38.0
+	go.opentelemetry.io/otel/exporters/prometheus v0.60.0
+	go.opentelemetry.io/otel/metric v1.38.0
+	google.golang.org/grpc v1.58.3
+	google.golang.org/protobuf v1.36.8
+	k8s.io/api v0.30.1
+	k8s.io/apimachinery v0.30.1
+	k8s.io/client-go v0.30.1
+	k8s.io/klog/v2 v2.120.1
 	k8s.io/mount-utils v0.26.4
-	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
+	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
 	monis.app/mlog v0.0.2
-	sigs.k8s.io/controller-runtime v0.14.6
+	sigs.k8s.io/controller-runtime v0.18.7
+)
+
+require (
+	github.com/google/gnostic-models v0.6.8 // indirect
+	github.com/grafana/regexp v0.0.0-20240518133315-a468a5bfb3bc // indirect
+	github.com/prometheus/otlptranslator v0.0.2 // indirect
+	go.opentelemetry.io/auto/sdk v1.1.0 // indirect
+	golang.org/x/exp v0.0.0-20220722155223-a9213eeb770e // indirect
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20230822172742-b8732ec3820d // indirect
 )
```

Comment:
We don't need a separate require block, could you move this to the `require` block below for indirect dependencies and run `go mod tidy`.

Reply:
Done